CRM/Core/Payment/PayPalImpl.php

```diff
@@ -793,23 +813,28 @@ public function doTransferCheckout(&$params, $component = 'contribute') {
       CRM_Core_Error::fatal(ts('Component is invalid'));
     }
 
-    $notifyURL = $config->userFrameworkResourceURL . "extern/ipn.php?reset=1&contactID={$params['contactID']}" . "&contributionID={$params['contributionID']}" . "&module={$component}";
+    $args = array(
```

Comment:
OK - well instead of changing the signature why not set a property on the parent class

$notifyURLVariables = array();

& then the parent function can include that.

Don't add paymentProcessorID though - that is already handled in the parent function